apps/server/src/modules/system/controller/system.controller.ts

```diff
@@ -5,12 +5,12 @@ import { SystemFilterParams } from '@src/modules/system/controller/dto/system.fi
 import { SystemOauthResponseMapper } from '@src/modules/system/controller/mapper/system-oauth-response.mapper';
 import { SystemUc } from '../uc/system.uc';
 
-@ApiTags('System')
-@Controller('system')
+@ApiTags('Systems')
+@Controller('systems')
 export class SystemController {
 	constructor(private readonly systemUc: SystemUc) {}
 
-	@Get()
+	@Get('public')
```

Comment:
for this exceptional case, where an endpoint is left public, it would be good to explain in code where/how is being used

Reply:
is it enough to add proper swagger docs? that should give enough context for this endpoint

Reply:
sure, that could work. A comment at least